photon-client/src/main/scala/com/linkedin/photon/ml/data/avro/ScoreProcessingUtils.scala

```diff
@@ -45,7 +45,7 @@ object ScoreProcessingUtils {
       val weight = Option(scoreAvro.getWeight).map(_.toDouble)
       val ids = scoreAvro.getMetadataMap.asScala.map { case (k, v) => (k.toString, v.toString) }.toMap
       val idsWithUid = uid match {
-        case Some(id) => ids + (AvroFieldNames.UID -> id)
+        case Some(id) => ids + (ResponsePredictionFieldNames.UID -> id)
```

Comment:
I think the `ScoreProcessingUtils` object is agnostic to the two different input schema types we have defined, so it should use `AvroFieldNames` instead. Regardless, it definitely shouldn't be using `ResponsePredictionFieldNames` as the `ScoringResultAvro` schema has the response field named "label", which would mean `TrainingExampleFieldNames` if not `AvroFieldNames`.